quilt/test/test_build.py

```diff
@@ -96,3 +97,49 @@ def test_failover(self):
         path = os.path.join(mydir, './build_failover.yml')
         build.build_package('test_failover', PACKAGE, path)
         from quilt.data.test_failover.groot import bad
+
+    def test_duplicates(self):
```

Comment:
Oh... I think it's because git doesn't allow empty directories. Let me fix that.